src/scss/objects/_form-toggle.scss

```diff
@@ -44,6 +47,10 @@ $formToggle-button-background       : $blue--offWhite;
         fill: $formToggle-button-color;
     }
 
+    .o-formToggle--inline {
```

Comment:
Always good to comment modifiers to say what they do when they are applied.  This one looks like it's adding spacing, but the name is a bit ambiguous as it implies it would make the formToggle inline?